client/components/recycleAndReuseComponents/MultiSelect.js

```diff
@@ -1,28 +1,58 @@
-import { Button } from "@chakra-ui/react";
-import React from "react";
-import Select from "react-select";
-import { options }  from '../../../mockData/data';
+import { Button } from '@chakra-ui/react'
+import React from 'react'
+import Select from 'react-select'
+import { options }  from '../../../mockData/data'
 
-const MultiSelect = ({ optionss = options, setInput }) => {
-  const handleChange = (event) => {
-    setInput({ input: event });
-  };
+const MultiSelect = ({ data, setItems, navigateToVerifyItem }) => {
+  const parseItemsIntoOptions = (categories, items) => {
+    const allOptions = []
+    categories.forEach(category => {
+      const itemsInCategory = items.filter(item => item.category === category)
+      const itemOptions = []
+      itemsInCategory.forEach(itemInCategory => {
+        const option = {
+          "label": itemInCategory.description,
+          "value": itemInCategory.id
+        }
+        itemOptions.push(option)
+      })
+
+      const categoryOptions = {
+        "label": category,
+        "options": itemOptions
+      }
+      allOptions.push(categoryOptions)
+    })
+    
+    return allOptions
+  }
+
+  const options = parseItemsIntoOptions(data.categories, data.items)
 
   const handleSubmit = (event) => {
-    event.preventDefault();
-    alert("submitting");
-  };
+    event.preventDefault()
+    const selectedItems = []
+    const selectedElements = event.target.elements
+    for (let i = 0; i < selectedElements.length; i++) {
+      if (selectedElements[i].nodeName === 'INPUT' && selectedElements[i].name === 'items') {
+        const selectedItem = data.items.filter(item => item.id === parseInt(selectedElements[i].value))
+        selectedItems.push(selectedItem[0])
+      }
+    }
+    setItems(selectedItems)
+    navigateToVerifyItem()
+  }
```

Comment:
On submit, we figure out what items have been submitted, set the items in the tabs state to these submitted items, and then automatically navigate to the next tab. Let me know if anything in this PR is hard to understand - I can add code comments.